tests/unit/test_vowpal_wabbit.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import pytest
+from unittest import mock
+
+
+import pandas as pd
+
+from reco_utils.recommender.vowpal_wabbit.vw import VW
+
+
+@pytest.fixture(scope="module")
+def df():
```

Comment:
Do we want to use the existing fixture for dummy data generation?

Reply:
we could but I still feel like this approach is a bandaid, i'd prefer to spend effort fixing the issue with vw python bindings themselves so we can remove this extra code.